cli/src/input.rs

```diff
@@ -66,6 +86,40 @@ impl<C: clap::Args + Customize> Prepare for InputOptions<C> {
             program.add_import_paths(nickel_path.split(':'));
         }
 
+        #[cfg(feature = "package-experimental")]
+        {
+            let manifest_path = self.manifest_path.clone().or_else(|| {
+                // If the manifest path isn't given, where should we start looking
+                // for it? If there's only one file, we start with its parent directory.
```

Comment:
When you say the parent directory, this means the directory where this file is located, not one the one above that, right?

Reply:
Right. I updated the comment to be clearer.